handbook/process/copyright.md

```diff
@@ -0,0 +1,37 @@
+The FPA can, if required by a contributor, assume the copyright over a contribution made by that contributor.
+
+The reason might be:
+
+1. **Give the FPA full control over the destiny of that file**
+
+The FPA and its administrators will then be able to do things like change the file's license without the author's consentment. It should be noted that this action concentrates copyright in the hands of a single body, which is not something advisable in a community-developed project. The FreeCAD code being in many hands is actually beneficial for the software, as it prevents a single actor from gaining too much power over the code. As protecting the community nature of FreeCAD is part of the statutes of the FPA, contributors are encouraged to not hand over their contributions to the FPA without a good reason. Their individual hold over part of the FreeCAD code's copyright is a strength to FreeCAD.
+
+2. **Protect the contributor against possible lawsuits**
+
+Contributors to FOSS projects may be subject to threats of lawsuits for copyright infringement. Projects bigger than FreeCAD have seen contributors threatened and scared enough to stop contributing to the project, which is the objective of the action. The FPA, being based in the European Union where software patents are inexistant, cannot be sued for patent infrigement. Individual contributors who live in countries subject to [software patents](https://en.wikipedia.org/wiki/Software_patent), such as the USA or Australia, might become a target of such threats.
+
+Any contributor thinking his contribution might be specific enough or touching domains where some commercial software specialize, and therefore might be seen as patent infringing, might choose to hand their copyright over to the FPA. This allows the FPA to claim they are the sole legal responsible for any harm this file might have caused, and deflect patent-infriging threats over itself.
+
+### Handing copyright over to the FPA
+
+Should a contributor decide to transfer their copyright to the FPA, the method to follow is simple, and no approval or requirement needs to be made against the FPA.
+They only need to amend (in case the contributor's name and email is already mentioned) or add (in case it is not), the copyright line found in every file's copyright header. For example:
+
+```
+Copyright (c) 2004 John Doe <john@doe.com>
+```
+
+Becomes:
+
+```
+Copyright (c) 2004 The FreeCAD project association <fpa@freecad.org>
+```
+
+If you are John Doe, or
```

Comment:
This example is confusing.  Was it meant to illustrate a joint copyright being amended to remove one person?

Reply:
Basically you don't remove other person's copyrights. You can just add your own